isis/src/lro/apps/lronac2pds/lronac2pds.h

```diff
@@ -0,0 +1,11 @@
+#ifndef lronac2pds_h
+#define lronac2pds_h
+
+#include "Pvl.h"
+#include "UserInterface.h"
+
+namespace Isis{
+  extern void lronac2pds(UserInterface &ui, Pvl *log=nullptr);
```

Comment:
The app doesn't seem to use a log? If so, probably best to remove the option from the api.